functional_test/src/test/java/com/newrelic/agent/instrumentation/ClassLoaderTest.java

```diff
@@ -72,7 +73,7 @@ public void testMissingNewRelicClass() throws ClassNotFoundException {
 
     // Java 12 no longer lets us access the declared field
     @Test
-    @Category({ Java12IncompatibleTest.class, Java13IncompatibleTest.class, Java14IncompatibleTest.class,
+    @Category({ IBMJ9IncompatibleTest.class, Java12IncompatibleTest.class, Java13IncompatibleTest.class, Java14IncompatibleTest.class,
             Java15IncompatibleTest.class, Java16IncompatibleTest.class, Java17IncompatibleTest.class, Java18IncompatibleTest.class })
     public void testSetSystemClassLoader() throws Exception {
 
```

Comment:
The `Classloader` class used in the J9 implementation does not have the field `scl`.